charts/velero-s3-deployment/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "1.0"
+appVersion: "RELEASE.2024-11-21T17-21-54Z"
```

Comment:
@pvannierop I remember we had an issue with s3-proxy application version in the helm chart, I don't remember what exactly it was. Do you think that issue will be present here as well?